Userland/Libraries/LibGfx/ClassicStylePainter.cpp

```diff
@@ -498,13 +500,19 @@ void ClassicStylePainter::paint_simple_rect_shadow(Painter& painter, IntRect con
     int corner_piece_height = min(half_height, base_size);
     int top_corners_bottom = base_size + corner_piece_height;
     int bottom_corners_top = base_size + max(half_height, sides_height - corner_piece_height);
+    auto vertical_corners_offset = border_radius > 0 ? base_size - (border_radius_offset - border_radius) : 0;
     auto paint_vertical = [&](int x, int src_row, int hshift, int hsrcshift) {
         Gfx::PainterStateSaver save(painter);
+        (void)hsrcshift;
```

Comment:
Can this be removed as a parameter now?

Reply:
Whoops, that's still used the `(void)` is just a leftover from me messing with the original code